cex-collection/go-example/main.go

```diff
@@ -21,10 +21,11 @@ import (
 )
 
 var (
-	PaymasterURL string
-	SponsorURL   string
+	SponsorURL string
+	ChainID    string
 
 	PolicyUUID                 uuid.UUID
```

Comment:
`PolicyUUID` is uselsee code, remove it from code and env.

Reply:
fixed, keep `PolicyUUID `  for js-sdk & go-sdk